dltxio/v2.getpaidinbitcoin.com.au
# GPIB-385

To test:
1. start client site and signup as nomal user and pass email and phone verify steps. expect add btc address show up,
   click generate HD address, expect HD address pop up and then move to next step, aslo can give btc address and click add address button and move to next step
2. sign up ad employee user and pass verify emial, expect add btc address show up , give btc address or generate HD address. expect move to next add payroll
